brushfire-features/src/main/scala/com/stripe/brushfire/features/DispatchedFeatureEncoder.scala

```diff
@@ -0,0 +1,232 @@
+package com.stripe.brushfire
+package features
+
+import scala.util.Try
+
+import com.twitter.algebird.{ Aggregator, Semigroup, Monoid }
+
+case class DispatchedFeatureEncoding[L](encoder: DispatchedFeatureEncoder)
+extends FeatureEncoding[String, FeatureValue, Map[L, Long]] with Defaults {
+  val splitter: Splitter[FeatureValue, Map[L, Long]] =
+    DispatchedSplitter(
+      BinarySplitter[Double, Map[L, Long]](LessThan(_)),
+      BinarySplitter[String, Map[L, Long]](EqualTo(_)),
+      doubleSplitter[Map[L, Long]],
+      SpaceSaverSplitter[String, L]()
+    )
+}
+
+object DispatchedFeatureEncoding {
+  import DispatchedFeatureEncoder.Counts
+
+  def trainer[L](
+    platform: TrainingPlatform
+  )(implicit
+    ev: platform.AggregateContext[Map[String, Option[Counts]]]
```

Comment:
Yeah - I'll take a look again and see if we can unify these things - or what the minimum is needed to build both on top of something else!